.env.example

```
@@ -36,6 +36,7 @@ INSTABUG_TOKEN='5c2d0f12fa12f9afc535585e5b7a9e79'
 MIXPANEL_TOKEN='0cb505dace6f4b3ceb9e17c7fcd7c66f'
 GCM_SENDER_ID='260468725946'
 BONUS_VACANZE_ENABLED=NO
+MYPORTAL_INTEGRATION_ENABLED=NO
```

could it be like this for shortness?
```suggestion
MYPORTAL_ENABLED=NO
```

---

Yes sounds good!

---

b3803a4 renames it!